src/workflows-app/components/InputsTable.test.ts

```diff
@@ -40,10 +40,13 @@ jest.mock('src/components/input', () => ({
 // mock out the height and width so that when AutoSizer asks for the width and height of "browser" it can use the mocked
 // values and render the component properly. Without this the tests will be break.
 // (see https://github.com/bvaughn/react-virtualized/issues/493 and https://stackoverflow.com/a/62214834)
-const originalOffsetHeight = Object.getOwnPropertyDescriptor(HTMLElement.prototype, 'offsetHeight');
-const originalOffsetWidth = Object.getOwnPropertyDescriptor(HTMLElement.prototype, 'offsetWidth');
+const originalOffsetHeight = Object.getOwnPropertyDescriptor(HTMLElement.prototype, 'offsetHeight')!;
+const originalOffsetWidth = Object.getOwnPropertyDescriptor(HTMLElement.prototype, 'offsetWidth')!;
```

Comment:
What does `!` at end mean? 🤔

Reply:
(or that we're asserting "it will definitely exist, so don't worry about null checks" to the typechecker)

Reply:
I assume this is a side effect of moving to ts, rather than anything specific about the minimum width? For future PRs it might be nice to do this in two commits so it's easy to review the main change and the refactor fixups separately